File: src/Services/WareHouse/WareHouse.Domain/Entity/AuditDetailSerial.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class AuditDetailSerial:BaseEntity
    {
        public string ItemId { get; set; }
        public string Serial { get; set; }
        public string AuditDetailId { get; set; }

        public virtual AuditDetail AuditDetail { get; set; }
        public virtual WareHouseItem Item { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/entity/AuditDetailSerial.ts
import { AuditDetail } from './AuditDetail';
import { WareHouseItem } from './WareHouseItem';
import { BaseEntity } from './BaseEntity';
import { Inward } from './Inward';
export interface AuditDetailSerial extends BaseEntity {
    itemId: string;
    serial: string;
    auditDetailId: string;
    auditDetail: AuditDetail;
    item: WareHouseItem;
}